static/src/javascripts/projects/common/modules/experiments/tests/habit-forming-digest-promo.js

```diff
@@ -0,0 +1,122 @@
+define([
+    'common/utils/config',
+    'common/utils/storage',
+    'common/utils/template',
+    'common/utils/fastdom-promise',
+    'common/modules/ui/message',
+    'common/modules/commercial/user-features',
+    'common/views/svgs',
+    'lodash/objects/defaults',
+    'lodash/arrays/uniq',
+    'text!common/views/experiments/habit-digest-promo.html'
+], function (
+    config,
+    storage,
+    template,
+    fastdomPromise,
+    Message,
+    userFeatures,
+    svgs,
+    defaults,
+    uniq,
+    digestPromo
+) {
+    return function () {
+        this.id = 'HabitFormingDigestPromo';
+        this.start = '2016-07-13';
+        this.expiry = '2016-08-13';
+        this.author = 'Kate Whalen';
+        this.description = 'Show infrequent users a banner offering a curated digest';
+        this.audience = 0;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Find if there is demand for various Guardian digest offerings';
+        this.audienceCriteria = 'Infrequent visitors, excluding first time visits';
+        this.dataLinkNames = 'habit forming digest promo';
+        this.idealOutcome = 'Visitors click on the CTA and demonstrate demand for the feature';
+
+        this.canRun = function () {
+            return !(config.page.isAdvertisementFeature) &&
+            config.page.contentType === 'Article' &&
+            isInfrequentVisitor() &&
+            hasSeenMembership();
+        };
+
+        var defaultData = {
+            arrowRight: svgs('arrowRight')
+        };
+
+        // check if the user is one of the target audience
+        function isInfrequentVisitor() {
+            if (storage.local.isStorageAvailable()) {
+                var alreadyVisited = storage.local.get('gu.alreadyVisited') || 0;
+                if (alreadyVisited > 3 && !userFeatures.isPayingMember()) {
```

Comment:
from Monica:

> Re audience, we will reverse analyse this afterwards, so of the people who see & interact with this, which fall into the non-regular / regular buckets of users etc as there is no good way to target those audiences at the moment.

so all we should be doing is:
* ensuring users only see the snap once
* ensuring we don't conflict with the membership snap

so it looks to me like this whole condition is irrelevant.